src/sidebar/components/TopBar.js

```diff
@@ -83,20 +82,24 @@ function TopBar({
         <span className="TopBar__login-links">⋯</span>
       )}
       {auth.status === 'logged-out' && (
-        <span className="TopBar__login-links">
-          <Button
-            className="TopBar__login-button"
-            buttonText="Sign up"
+        <span className="TopBar__login-links u-font--large u-horizontal-rhythm">
```

Comment:
Using `u-font--large` here to establish the content sizing of the following `LinkButton`s